web/yo/app/scripts/directives/reviewpanel.js

```diff
@@ -232,6 +243,26 @@ angular.module('oncokbApp')
                             $scope.tm.trials.pushAll($scope.tm.trials_review.get('lastReviewed'));
                             rejectItem([{reviewObj: $scope.tm.trials_review, uuid: $scope.tm.trials_uuid}]);
                             break;
+                        case 'MUTATION_NAME_CHANGE':
+                            rejectItem([{obj: $scope.mt.name, reviewObj: $scope.mt.name_review, uuid: $scope.mt.name_uuid}]);
+                            break;
+                        case 'TUMOR_NAME_CHANGE':
+                            var lastReviewed = $rootScope.model.createList();
+                            _.each($scope.tm.cancerTypes_review.get('lastReviewed'), function(ct) {
```

Comment:
actually not necessary. There is no problem for me to use it this way. Probably because _ is the default reference for underscore.js, which we already included in the index.html